packages/utils/src/chart.js

```diff
@@ -0,0 +1,33 @@
+export const CHART_TYPES = {
+  AREA: 'area',
+  BAR: 'bar',
+  COMPOSED: 'composed',
+  LINE: 'line',
+  PIE: 'pie',
+  GAUGE: 'gauge',
+};
+
+export const getIsChartData = ({ chartType, data }) => {
```

Comment:
Can we please move  `CHART_TYPES ` and `getIsChartData ` to the charts package.

Overall the move of the utils from chart to utils looks great. I think these 2 should belong in charts though.